<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientManagerDlg</class>
 <widget class="QDialog" name="ClientManagerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>单位名称：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="clientNameBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>地址：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="clientAddrEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="contactsGroup">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>添加联系人</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="6">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>联系电话：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>职务：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>联系人：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4" colspan="2">
       <widget class="QLineEdit" name="postEdit"/>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="contactsEdit"/>
      </item>
      <item row="1" column="7">
       <widget class="QLineEdit" name="phoneEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>电子邮箱：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QPushButton" name="addContactsBtn">
        <property name="text">
         <string>增加</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="6">
       <widget class="QLineEdit" name="eMailEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="MyTableView" name="contactsView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="OKButton">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTableView</class>
   <extends>QTableView</extends>
   <header location="global">mytableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
